src/lib/datepicker/multi-year-view.ts

```diff
@@ -132,7 +132,20 @@ export class MatMultiYearView<D> implements AfterContentInit {
   _init() {
     this._todayYear = this._dateAdapter.getYear(this._dateAdapter.today());
     let activeYear = this._dateAdapter.getYear(this._activeDate);
+
+    // Default Behavior for Offset
     let activeOffset = activeYear % yearsPerPage;
+
+    if (!this._maxDate && this._minDate) {
+      activeOffset = 0;
+    }
+
+    if (!this._minDate && this._maxDate || this._minDate && this._maxDate) {
+      let yearOffset = activeYear - this._dateAdapter!.getYear(this._maxDate);
```

Comment:
Prefer `const` over `let`. Also I don't think you need the `!` after `_dateAdapter`